idea/frontend/src/features/voucher/ui/issue-voucher-modal/issue-voucher-modal-deprecated.tsx

```diff
@@ -40,7 +40,8 @@ const IssueVoucherModalDeprecated = ({ programId, close }: Props) => {
     const decodedAddress = decodeAddress(address);
     const chainValue = getChainBalanceValue(value).toFixed();
 
-    issueVoucherDeprecated(decodedAddress, programId, chainValue, close);
+    // voucher without specified programId is not supported in deprecated versions
+    issueVoucherDeprecated(decodedAddress, programId!, chainValue, close);
```

Comment:
I think we can remove supporting old voucher since we switched to 1.1.0 on both Vara and Testnet networks

Reply:
I'll do it in a standalone PR, there are quite a lot of code to remove.